### DISCUSSION TITLE
Support for FCCID https://fccid.io/EWQ500GA ?

### DISCUSSION THREAD

**Discussion by digiblur** [bemasher/rtlamr#205] [Q&A]
I noticed the gas meters in the area are using these modules.  Anyone have experience with them?  I've done a bit of searching but not sure the frequency they are using given the range on the FCC docs seem to cover the entire 900mhz ISM band.  

https://fccid.io/EWQ500GA

**Comment by bmcdonough** [comment]
I am confirming ITRON OPENWAY RIVA 500G, ERG-7000-001 with FCC ID EWQ500GA is decoded:

- window 1:
```shell
rtl_tcp
```

- window 2:
```shell
rtlamr -msgtype=scm+ -format=json
```

- results, I obfuscated my meter id.
```json
{"Time":"2026-05-18T13:28:49.618761078Z","Offset":0,"Length":49152,"Type":"SCM+","Message":{"FrameSync":5795,"ProtocolID":30,"EndpointType":188,"EndpointID":123456789,"Consumption":14924,"Tamper":5128,"PacketCRC":3415}}
```

@bemasher , is there any additional information needed to update the meters.csv?

**Comment by bmcdonough** [comment]
additional details provided by rtl_433, which also decodes:
- shell
```shell
rtl_433 -d 2 -f 912600155 -s 2359296 -M level -F json
```
- results, meter id obfuscated
```json
{"time" : "2026-05-18 13:47:04", "model" : "SCMplus", "id" : 103454546, "ProtocolID" : "0x1E", "EndpointType" : "0xBC", "EndpointID" : 123456789, "Consumption" : 14926, "Tamper" : "0x1408", "PacketCRC" : "0x6337", "MeterType" : "Gas", "mic" : "CRC", "mod" : "ASK", "freq" : 912.578, "rssi" : -3.945, "snr" : 24.582, "noise" : -28.527}
```
